web/src/TokenListPage.js

```diff
@@ -26,18 +26,24 @@ class TokenListPage extends React.Component {
     this.state = {
       classes: props,
       tokens: null,
+      total: 0,
+      page: 1,
+      pageSize: 5,
```

Comment:
one state is enough

Reply:
only save total? If that, there will be no loading effect.